### PR TITLE
adding jest config, need to define on webstorm --watch

### PR DESCRIPTION
**Pull request by yinonc**

### CODE REVIEW

**Comment by yinonc** [comment]
@stavalfi Configuration for watch: http://www.interload.co.il/upload/3285200.png
In order to use "debug" script - for a specific file test, add `debugger` statement in your test, and add on Chrome: https://chrome.google.com/webstore/detail/nodejs-v8-inspector-manag/gnhhdgbaldcilmgcpfddgdbkhjohddkj

Usage:
1. Copy working file path (CMD + Shift + C shortcut on webstorm)
2. run `yarn debug` + paste filePath and paste 
Very useful! Chrome debugging is the best

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "start": "cross-env NODE_ENV=development webpack-serve",
     "build": "cross-env NODE_ENV=deployment webpack",
     "eslint": "eslint src --fix",
-    "test": "node $NODE_DEBUG_OPTION node_modules/.bin/ava"
+    "test1": "node $NODE_DEBUG_OPTION node_modules/.bin/ava",
```
Comment (stavalfi) [inline package.json]: please remove any ava configuration, files, and dependencies from the project.

File: package.json [hunk 2]
```diff
@@ -41,6 +44,9 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0"
   },
+  "resolutions": {
```
Comment (stavalfi) [inline package.json]: We already have babel/core under devDependencies.
https://github.com/stavalfi/workflow/blob/c5ecb730a7c506769c71e02b19e53ff531d1dc1f/package.json#L19

**Comment by stavalfi** [comment]
cool. just one little thing:  remove any ava configuration, files, and dependencies from the project.

File: package.json (hunk 2 above)
Comment (yinonc) [reply package.json]: https://github.com/facebook/jest/issues/6913

File: package.json (hunk 1 above)
Comment (yinonc) [reply package.json]: done

**Comment by stavalfi** [comment]
you did that already haha.. cool

**Comment by yinonc** [comment]
@stavalfi When you'll have a readme I'll add the whole usage of tests

File: package.json (hunk 2 above)
Comment (stavalfi) [reply package.json]: then remove "@babel/core": "^7.1.5",  and install "babel-core": "7.0.0-bridge.0" instead under devDependencies ? or is it a problem also? 

Choose any solution you want but try to remove one of them.

**Comment by stavalfi** [comment]
great.

**Comment by yinonc** [comment]
@stavalfi please approve it I don't have access ATM

**Comment by stavalfi** [comment]
there are 2 versions of babel/core in package.json. remove one of them.

**Comment by yinonc** [comment]
@stavalfi done

**Pull request merged**